test/SampleStream.test.js

```diff
@@ -29,10 +20,10 @@ function testFunc(r, f, expected, t) {
     });
 
     for (var k = 0; k < 1000; k++) {
-    // each of these is 9 records long
-        transformedStream.write({Body: fs.readFileSync(path.join(__dirname + '/fixtures/AAAAAAAAAAAAAA.2015-10-19-17.e5b6526a.gz'))});
+        // each of these is 9 records long
+        splitStream.write(fs.readFileSync(path.join(__dirname + '/fixtures/AAAAAAAAAAAAAA.2015-10-19-17.e5b6526a'), 'utf8'));
     }
-    transformedStream.end();
+    splitStream.end();
```

Comment:
This was the part it took me a bit to figure out. I always forget that these two things are different:

```
var stream = split().pipe(sample);

// This will write to the `sample` stream!
stream.write( ... );
```

vs

```
var stream = split();
stream.pipe(sample);

// This will write to the `split` stream!
stream.write( ... );
```

In this case we want to write our raw log fixture to the split stream, and let it break things into individual log line chunks to be piped to the sampler.